recipes/libprotobuf/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+
+if [ "$(uname)" == "Darwin" ];
+then
+    # Switch to clang with C++11 ASAP.
+    export MACOSX_VERSION_MIN=10.7
+    export CC=clang
+    export CXX=clang++
+    export CXXFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11 -O2"
+    export LIBS="-lc++"
```

Comment:
This has deviated from https://github.com/conda-forge/toolchain-feedstock/blob/master/recipe/activate.sh
I'd suggest using toolchain, but some people prefer having it explicitly in the recipe.

Reply:
Added PR ( https://github.com/conda-forge/protobuf-feedstock/pull/44 ) to correct `protobuf`.